src/data/colleges/en.ts

```diff
@@ -10,22 +10,40 @@ import {
 import { AdvisorGroup } from '../../tools/advisors/types';
 import { lastNameRange, lastNameRanges } from '../../tools/advisors/checkers';
 
-const engineeringLiberalArtsDistributions: readonly string[] = [
-  'CA',
-  'HA',
-  'LA',
-  'LAD',
-  'KCM',
-  'SBA',
-  'FL',
-  'CE',
-  'ALC',
-  'SCD',
-  'HST',
-  'ETM',
-  'SSC',
-  'GLC',
-];
+const engineeringLiberalArtsGroups: Record<string, string[]> = {
```

Comment:
Thanks for the hard work migrating all of these categories manually! And I agree, a likely cause of this issue in the first place was poor wording with groups x categories (our fault or the school's 🤔). Good catch!